website/src/components/ask-ai/AISearch.jsx

```diff
@@ -78,7 +78,7 @@ const AISearch = forwardRef((props, ref) => {
         registerAISearch(query);
 
         try {
-            const apiUrl = "https://hook.eu1.make.com/zw41brw94vfiwuue91riqmoe2ms2nhf6";
+            const apiUrl = "https://ai.appsmith.com/api/v1/docs/query";
```

Comment:
The generated response from this API is probably not in Markdown format and hence the styling is broken. For example, the code blocks.
Below is the screenshot-
<img width="803" alt="GitBranch" src="https://github.com/appsmithorg/appsmith-docs/assets/99252011/e38811f0-822f-42da-a289-451cd1685851">
<img width="818" alt="prod-docs" src="https://github.com/appsmithorg/appsmith-docs/assets/99252011/f23a5893-aa73-476a-89a3-63012eda5e8f">

Reply:
Do you see the wrong formatting every time you try it? This is what I got now.

<img width="938" alt="Screenshot 2024-03-06 at 11 42 36 AM" src="https://github.com/appsmithorg/appsmith-docs/assets/18716465/6492960b-440e-4893-8d7e-6456581d01a3">

Reply:
I tried again, but this time it is not giving formatting issues. Do we know why it breaks sometimes?

Reply:
If the formatting issues are consistent, then it is usually a problem with the instruction. I have updated the instruction to handle this.